src/sentry/receivers/onboarding.py

```diff
@@ -259,6 +260,19 @@ def record_first_feedback(project, **kwargs):
     )
 
 
+@first_new_feedback_received.connect(weak=False)
+def record_first_new_feedback(project, **kwargs):
+    project.update(flags=F("flags").bitor(Project.flags.has_new_feedbacks))
```

Comment:
assume its 500 because this is against master and needs the migration for this line to work